src/HealthChecks.MySql/MySqlHealthCheck.cs

```diff
@@ -13,10 +13,6 @@ public class MySqlHealthCheck : IHealthCheck
     public MySqlHealthCheck(MySqlHealthCheckOptions options)
     {
         Guard.ThrowIfNull(options);

```

Comment:
Since we started the refactoring, can we simplify it even further?

```suggestion
        _options = Guard.ThrowIfNull(options);
```